libs/libwebsockets/Makefile

```diff
@@ -60,10 +60,11 @@ define Package/libwebsockets-openssl
 endef
 
 define Package/libwebsockets-mbedtls
-	$(call Package/$(PKG_NAME)/Default)
```

Comment:
I am thinking if I should not split this change to its own commit

Reply:
Personally, I don't think it should be changed at all, it's just style. :)

Reply:
That's true. It should be in sync with others, but let's drop this. Unrelated to my change.